charts/kubewarden/1.1.1-rc1/templates/cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: catalog.cattle.io/v1
+kind: UIPlugin
+metadata:
+  name: {{ include "extension-server.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "extension-server.labels" . | nindent 4 }}
+spec:
+  plugin:
+    name: {{ include "extension-server.fullname" . }}
+    version: {{ (semver (default .Chart.AppVersion .Values.plugin.versionOverride)).Original }}
+    endpoint: https://raw.githubusercontent.com/kubewarden/kubewarden-ui/gh-pages/extensions/kubewarden/1.1.0-rc1
```

Comment:
The endpoint should be pointing to `1.1.1-rc1`

Reply:
didn't realize about that. I've templated it with `{{ .Chart.AppVersion }}` in https://github.com/kubewarden/kubewarden-ui/pull/441/commits/456f95ac350f051f82230e113434185bf2268fdc, and tested it with `helm template charts/kubewarden/1.1.1-rc1`.

I think templating this should be ok, no? the chart will always need an endpoint matching its version, correct?

Reply:
I think that should be fine, it will need to match the version but this will certainly provide the correct version.